aya/src/obj/mod.rs

```diff
@@ -873,6 +922,22 @@ mod tests {
         }
     }
 
+    fn fake_sym(obj: &mut Object, section_index: usize, name: &str, size: u64) {
+        let idx = obj.symbols_by_index.len();
+        obj.symbols_by_index.insert(
+            idx + 1,
+            Symbol {
+                index: idx + 1,
+                section_index: Some(section_index),
+                name: Some(name.to_string()),
+                address: 0,
```

Comment:
The tests are parsing only the first map - you need to set the `address` field now